### DISCUSSION TITLE
Apply compression to deform a generated image of porous media?

### DISCUSSION THREAD

**Discussion by josephinoo** [PMEAL/porespy#533] [Q&A] [answered]
Hello, It is possible to add parameters such as pressure, and compression to porous media. if so. How could one add these generated porous mean parameters?

**Comment by jgostick** [comment, marked as answer]
I think you're asking if it's possible to squeeze an image to compress it?  This extremely difficult since it requires doing a finite element simulation, and is not possible with porespy.  There is a commercial software called geodict that can do this sort of thing.  There are some free packages, such as yade, that let you do physics based packing of particles.
